library/registry

```diff
@@ -1,7 +1,23 @@
-Maintainers: David Wu (@davidswu),
-             Ryan Abrams (@caervs)
+Maintainers: Chris Patterson (@chrispat),
+             Bryan Clark (@clarkbw),
+             Cory Snider (@corhere),
+             Hayley Swimelar (@deleteriousEffect),
+             He Weiwei (@heww),
+             João Pereira (@joaodrp),
+             Justin Cormack (@justincormack),
+             Kyle Squizzato (@squizzi),
+             Milos Gajdos (@milosgajdos),
+             Sargun Dhillon (@sargun),
+             Wayne Warren (@waynr),
+             Wang Yan (@wy65701436),
+             Steve Lasker (@stevelasker)
```

Comment:
Did you mean to add this full list?  This should be just the maintainers of the image repository (ie, https://github.com/docker/distribution-library-image), not the full list of maintainers of the upstream software. :sweat_smile:

(We use these for things like automated notifications to existing maintainers when an image is updated or to ping folks on issues like the Alpine version you're based on going away, etc.)

Reply:
Ohhhh, I see! Well, it's.......complicated :)
But for this release, I guess it can be just me and @thaJeztah 

In the future the image repo will move under `distribution` org -- in which case it'll have to be either all maintainers or some select group hehe

Reply:
https://github.com/docker-library/official-images/pull/11718/commits/6f33409389c501db767c7da48ed1a3f5afcbea5a

Reply:
Heh, fair enough -- we'd also be happy to maintain it under the @docker-library org if you'd like (just to make sure you know you've got options)